app/assets/images/building.svg

```diff
@@ -1,5 +1,5 @@
 <!-- Icon from noun project -->
-<svg width="19" height="22" viewBox="0 0 19 22" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path class="fill-current text-pink-600" d="M17.3297 2.76235H0.752999L0.0627441 0H18.02L17.3297 2.76235Z" />
-<path class="fill-current text-pink-600" d="M1.44336 4.1442V22.1028H7.65963V17.9586H10.422V22.1028H16.6383L16.6395 4.1442H1.44336ZM6.27912 16.5767H3.51678V13.8144H6.27912V16.5767ZM6.27912 12.4326H3.51678V9.67021H6.27912V12.4326ZM6.27912 8.28838H3.51678V5.52603H6.27912V8.28838ZM10.4233 16.5767H7.66096V13.8144H10.4233V16.5767ZM10.4233 12.4326H7.66096V9.67021H10.4233V12.4326ZM10.4233 8.28838H7.66096V5.52603H10.4233V8.28838ZM14.5675 16.5767H11.8051V13.8144H14.5675V16.5767ZM14.5675 12.4326H11.8051V9.67021H14.5675V12.4326ZM14.5675 8.28838H11.8051V5.52603H14.5675V8.28838Z" />
```

Comment:
Dette for å kunne bruke ikonet andre steder også, i andre farger. 

Vi vil måtte sette text-pink-600 på der hvor ikonet brukes nå, men det gjøres ikke i denne branchen atm.